src/main/webapp/WEB-INF/themes/desktop/sass/sections/_header.scss

```diff
@@ -115,22 +115,8 @@ $width-nav: $block-grid-elements - $width-logo;
 #pagehdr .nav-main .logo {
   position: absolute;
   float: none;
-  margin-left: .625rem;
```

Comment:
@samV-plos Do you want the logos to appear left-flush with the article content or do you want to retain some margin?

Reply:
I prefer left-aligned.  Just as long as the article pages are left-aligned /flush as well.

Reply:
excellent, thanks @samV-plos !